### DISCUSSION TITLE
GFM: collapsible sections not working

### DISCUSSION THREAD

**Discussion by vforvoid** [alshedivat/al-folio#1236] [Q&A] [answered]
Hi there. 

First of all, I want to say thank you for all the great work you did creating this theme.
I just started kicking the tires here and was wondering if the fact that [GFM collapsed sections](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections) not working is expected behavior or a bug. It's not a part of a standard markdown and maybe you don't have any intention to support it.

I put this code into my post markdown file:
````Markdown
<details><summary>CLICK ME</summary>
<p>

#### We can hide anything, even code!

```ruby
   puts "Hello World"
```

</p>
</details>
````

And it got rendered like this in HTML:
![image](https://user-images.githubusercontent.com/14206129/222974512-9e544f09-92fe-4372-949b-1424fb2a287c.png)

**Comment by george-gca** [comment, marked as answer]
The easiest way I found for doing this is using the `highlight` liquid tag instead of pure markdown. Like this:

```html
<details><summary>CLICK ME</summary>
<p>

#### We can hide anything, even code!

{% highlight ruby %}
   puts "Hello World"
{% endhighlight %}

</p>
</details>
```
